web/cmd/server/main.go

```diff
@@ -8,12 +8,13 @@ import (
 	// registers all components.
 	_ "go.viam.com/rdk/components/register"
 	"go.viam.com/rdk/logging"
+
 	// registers all services.
 	_ "go.viam.com/rdk/services/register"
 	"go.viam.com/rdk/web/server"
 )
 
-var logger = logging.NewDebugLogger("rdk")
+var logger = logging.NewDebugLogger("entrypoint")
```

Comment:
These `rdk` loggers were colliding with the `rdk` logger in `entrypoint.go` since they were named the same, but since these seem to be mostly responsible with logging issues with rdk startup (and there already exists a `startup`), I thought `entrypoint` would be more fitting

Reply:
Yeah `entrypoint` here seems good.